lua/core/comment.lua

```diff
@@ -0,0 +1,24 @@
+local M = {}
+
+M.config = function()
+  lvim.builtin.comment = {
+    active = true,
+    on_config_done = nil,
```

Comment:
You don't need to set it to nil as it's the default value of missing table entries

Reply:
Unless you want to make it explicit

Reply:
The idea is to provide easier discover-ability for users using the lua language server. I'd prefer to convey the signature as well, but couldn't find a good solution for it.